src/testRunner/unittests/tscWatch/forceConsistentCasingInFileNames.ts

```diff
@@ -295,6 +295,8 @@ a;b;
             }),
             "/Users/name/projects/web/tsconfig.json": JSON.stringify({
                 compilerOptions: {
+                    target: "es5",

```

Comment:
Added target: es5 to these because `module: nodenext` implies `target: esnext` which changes the default lib file to one that is not set up to exist for these tests.